Tools/IoTConnectionTools/ibm-device/cpp/deviceSample.c

```diff
@@ -0,0 +1,244 @@
+/*******************************************************************************
```

Comment:
As far as I can tell, this was not previously disclosed as a dependency. Please inform Swapna that this sample will require OSPDT approval for deviceSimple

Reply:
> As far as I can tell, this was not previously disclosed as a dependency. Please inform Swapna that this sample will require OSPDT approval for deviceSimple

The IBM Watson IoT Platform C Client SDK is approved as 'Conditional' (Slide 11 of Open Source PDT v7.0 - oneAPI samples Beta09.pptx). This sample (deviceSample) is a part of the mentioned approved library.

Reply:
Thank you for the clarification